packages/form-js-editor/src/features/properties-panel/entries/StaticValuesSourceEntry.js

```diff
@@ -29,7 +29,7 @@ export default function StaticValuesSourceEntry(props) {
     editField(field, VALUES_SOURCES_PATHS[VALUES_SOURCES.STATIC], without(values, entry));
   };
 
-  const validateFactory = (key) => {
+  const validateFactory = (key, map) => {
```

Comment:
I think it's a bit more clear

```suggestion
  const validateFactory = (key, getValue) => {
```